.github/workflows/builder_installation_assistant.yml

```diff
@@ -0,0 +1,90 @@
+run-name: Build Installation Assistant ${{ inputs.id }} - Wazuh installation assistant branch ${{ github.ref_name }} - Launched by @${{ github.actor }}
```

Comment:
```suggestion
run-name: Build Installation Assistant ${{ inputs.id }} - Branch ${{ github.ref_name }} - Launched by @${{ github.actor }}
```

Reply:
Fixed in https://github.com/wazuh/wazuh-installation-assistant/commit/aa18f6c81eb4aae8d54dab76e8dfdb18d651545e